src/components/Pricing/PricingExperiment.tsx

```diff
@@ -0,0 +1,1326 @@
+
+                        <div>Volume limits</div>
+                        <div className="col-span-2">
+                            Limited to free monthly allowance{' '}
+                            <Tooltip
+                                content={() => (
+                                    <div className="max-w-[250px]">
+                                        Usage is capped at the free tier limits for each product and resets monthly.
+                                    </div>
+                                )}
+                                placement="top"
+                            >
+                                <IconInfo className="size-4 inline-block" />
+                            </Tooltip>
+                        </div>
+                        <div className="col-span-2">Unlimited</div>
+
+                        <div>Product features</div>
+                        <div className="col-span-2">
+                            Basic features{' '}
+                            <Tooltip
+                                content={() => (
+                                    <div className="max-w-[320px]">
+                                        <p className="mb-2 text-sm">
+                                            Use each product for free without advanced features. Compare functionality
+                                            limitations on the product page.
+                                        </p>
+                                        <p className="mb-0 text-sm">
+                                            For full functionality, just enter a credit card and you'll be on the{' '}
+                                            <em>Ridiculously cheap</em> plan. PostHog is still free up to the monthly
+                                            free tier limits, and you can set a billing limit as low as $0.
+                                        </p>
+                                    </div>
+                                )}
+                                placement="top"
+                            >
+                                <IconInfo className="size-4 inline-block" />
+                            </Tooltip>
+                        </div>
+                        <div className="col-span-2">All features</div>
+
+                        <div>Support</div>
+                        <div className="col-span-2">Community forums</div>
+                        <div className="col-span-2">Email</div>
+
+                        <div>
+                            Add-ons{' '}
+                            <Tooltip
+                                content={() => (
+                                    <div className="max-w-[300px]">
+                                        Add-ons extend functionality of products, but all are billed per use and as
+                                        such, not available on the <em>Totally free</em> plan.
```

Comment:
The Person profiles addon will be available on the free plan.. The 1M events on the free plan can be with or without person profiles.